pt_BR locale update

mtub opened the pull request:
Updated pt_BR locale files based on locale files provided by Ramón (see http://forum.pkp.sfu.ca/t/how-to-submit-translation-for-review/9033 and https://pkp.sfu.ca/wiki/index.php?title=OJS:_Portuguese_%28pt_BR%29)

mtub commented:
@asmecher , can you take a quick look?

asmecher commented on plugins/blocks/customnavbar/locale/pt_BR/locale.xml:
```diff
@@ -0,0 +1,18 @@
+<!--
+  * plugins/blocks/customnavbar/locale/pt_BR/locale.xml
+  *
+  * Copyright (c) 2013-2014 Simon Fraser University Library
```
Old copyright dates. (Also below)

mtub commented:
Removed the locale files with wrong copyright date - they didn't belong in here. While on the task, fixed another wrong copyright date and removed another orphaned locale file.

asmecher commented on plugins/importexport/crossref/locale/pt_BR/locale.xml:
```diff
@@ -13,31 +13,38 @@
-	<message key="plugins.importexport.crossref.export.unregistered">Artigos não registrados</message>
-	<message key="plugins.importexport.crossref.export.selectUnregistered">Selecionar artigos não registrados</message>
-	<message key="plugins.importexport.crossref.export.noUnregistered">Todos os artigos publicados já foram registrados (ou nenhum deles têm um DOI designado).</message>
+	<message key="plugins.importexport.crossref.description">Export or register article metadata in CrossRef format.</message>
```
This swaps the pt_BR out for en_US text.

asmecher commented on plugins/metadata/dc11/locale/pt_BR/locale.xml:
```diff
@@ -0,0 +1,17 @@
+  -->
+
+<locale name="pt_BR" full_name="Português (Brasil)">
+	<message key="plugins.metadata.dc11.articleAdapter.displayName"></message>
```
The added strings here are empty.

asmecher commented on plugins/paymethod/paypal/locale/cs_CZ/locale.xml:
```diff
@@ -4,8 +4,8 @@
-  * Copyright (c) 2014-2016 Simon Fraser University Library
-  * Copyright (c) 2006-2014 Gunther Eysenbach, Juan Pablo Alperin, MJ Suhonos
+  * Copyright (c) 2013-2016 Simon Fraser University Library
+  * Copyright (c) 2006-2009 Gunther Eysenbach, Juan Pablo Alperin, MJ Suhonos
```
I'm nervous about rolling back the dates for non-team-members; best to leave this as it was.

mtub replied:
But this is just setting the copyright dates in cs_CZ to those in every other localized version of this file.

asmecher replied:
Hmm, not sure where that change got in, then. OK, leave it as the PR has it.

mtub replied:
To clarify: no other file but this has a copyright longer than 2009 for the non-team contributors. This is what the en_US original file looks like. Please decide - I can revert this, but I don't think it's necessary.

asmecher replied:
I've maintained the en_US copyright dates reasonably carefully; it's most likely that this was changed by a 3rd party. It's best to stay consistent with what en_US has.

mtub commented:
everything but the copyright date issue (see above) should be resolved now.

asmecher commented:
Thanks, @mtub! Merged.
